app/src/main/java/com/sun/uefascore/utils/ComponentExt.kt

```diff
@@ -0,0 +1,10 @@
+package com.sun.uefascore.utils
+
+import android.os.AsyncTask
+import android.widget.ImageView
+
+fun ImageView.getImageUrl(url: String) {
```

Comment:
```suggestion
fun ImageView.loadImageUrl(url: String) {
```

file CommonExt chứ không phải ComponentExt

Reply:
done